### ISSUE TITLE
FileUploader: Value State Message is not being announced by the Screen Reader

### ISSUE THREAD

**Issue by achuth2912** [UI5/webcomponents-react#8603]
### Describe the bug

When we upload a file using the file uploader component the valueStateMessage is not being announced by the File Uploader component.

### Isolated Example

https://stackblitz.com/edit/github-xrgckykb?file=src%2FApp.tsx

### Reproduction steps

1. Upload a File
2. Screen reader does not announce the valueStateMessage
3.
...

### Expected Behaviour

Screen reader must announce the valueStateMessage on upload.

### Screenshots or Videos

<img width="1728" height="955" alt="Image" src="https://github.com/user-attachments/assets/921ad660-4a2e-4e76-8dc1-bcf3f7215250" />

### UI5 Web Components for React Version

2.19.2

### UI5 Web Components Version

2.19.2

### Browser

Chrome

### Operating System

Windows, Mac

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
